mj_envs/hand_manipulation_suite/pen_v0.py

```diff
@@ -118,15 +127,15 @@ def ss(self, state_dict):
 
     def mj_viewer_setup(self):
         self.viewer = MjViewer(self.sim)
-        self.viewer.cam.azimuth = 90
+        self.viewer.cam.azimuth = -45
         self.sim.forward()
-        self.viewer.cam.distance = 1.2
+        self.viewer.cam.distance = 1.0
 
     def evaluate_success(self, paths):
         num_success = 0
         num_paths = len(paths)
         for path in paths:
-            if np.sum(path['rewards']) > 3000:
+            if np.sum(path['env_infos']['goal_achieved']) > 20:
```

Comment:
add a comment explaining what you are considering as success.

Reply:
Added a commit to address this.